_includes/form-success.html

```diff
@@ -1,5 +1,9 @@
 <!-- <div id="form-success" class="form-success-hidden"> -->
 <div id="form-success" class="form-success-hidden alert alert-success alert-dismissible" role="alert">
   <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-  <h3>Thank you. Your request was successfully submitted</h3>
+  {% if page.url == "/manage-apps/register.html" %}
```

Comment:
Now that you mention it, title: Sandbox Registration already exists. I guess I could use that.